cmd/networkPolicies.go

```diff
@@ -3,50 +3,185 @@ package cmd
+	if len(netpol.Spec.Egress) > 0 {
+		return false
+	}
+
+	return true
+}
+
+func checkNamespaceNetworkPolicies(netPols *NetworkPolicyListV1, result *Result) {
+	// TODO check if any netpol is default Policy
+	hasDefaultDeny := false
 
 	for _, netPol := range netPols.Items {
+		// If not set check if default deny policy
+		if !hasDefaultDeny {
+			hasDefaultDeny = checkIfDefaultDenyPolicy(netPol)
+		}
+
 		for _, ingress := range netPol.Spec.Ingress {
+			// Allow all ingress traffic
 			if (len(ingress.From)) == 0 {
-				log.WithField("type", "netpol").
-					Warn("Default allow mode on ", netPol.Namespace, "/", netPol.Name)
+				occ := Occurrence{
+					container: "",
+					id:        WarningAllowAllIngressNetworkPolicyExists,
+					kind:      Warn,
+					message:   "Found allow all ingress traffic NetworkPolicy",
+					metadata: Metadata{
+						"PolicyName": netPol.ObjectMeta.Name,
+					},
+				}
+				result.Occurrences = append(result.Occurrences, occ)
 			}
 		}
+
+		for _, egress := range netPol.Spec.Egress {
+			// Allow all egress traffic
+			if (len(egress.To)) == 0 {
+				occ := Occurrence{
+					container: "",
+					id:        WarningAllowAllEgressNetworkPolicyExists,
+					kind:      Warn,
+					message:   "Found allow all egress traffic NetworkPolicy",
+					metadata: Metadata{
+						"PolicyName": netPol.ObjectMeta.Name,
+					},
+				}
+				result.Occurrences = append(result.Occurrences, occ)
+			}
+		}
+	}
+
+	if !hasDefaultDeny {
+		occ := Occurrence{
+			container: "",
+			id:        ErrorMissingDefaultDenyNetworkPolicy,
+			kind:      Error,
+			message:   "Namespace is missing a default deny NetworkPolicy",
+		}
+		result.Occurrences = append(result.Occurrences, occ)
+	} else {
+		occ := Occurrence{
+			container: "",
+			id:        InfoDefaultDenyNetworkPolicyExists,
+			kind:      Info,
+			message:   "Namespace has a default deny NetworkPolicy",
+		}
+		result.Occurrences = append(result.Occurrences, occ)
 	}
 
-	if len(badNetPols) != 0 {
-		for _, netPol := range badNetPols {
-			log.WithField("type", "netpol").Error(netPol.Name)
+	return
+}
+
+func getNetworkPoliciesResources(namespace string) (netPolList *NetworkPolicyListV1, err error) {
+	// Prevent the return of a nil value
+	netPolList = &NetworkPolicyListV1{}
+	if rootConfig.manifest != "" {
+		resources, err := getKubeResourcesManifest(rootConfig.manifest)
+		if err != nil {
+			return netPolList, nil
+		}
+
+		for _, resource := range resources {
+			switch kubeType := resource.(type) {
+			case *NetworkPolicyV1:
+				netPolList.Items = append(netPolList.Items, *kubeType)
+			}
 		}
+
+		return netPolList, nil
+	}
+
+	currentRootNamespace := rootConfig.namespace
+	kube, err := kubeClient()
+	if err != nil {
+		return netPolList, err
 	}
+
+	if namespace != "" {
+		rootConfig.namespace = namespace
+	}
+
+	netPolList = getNetworkPolicies(kube)
+
+	rootConfig.namespace = currentRootNamespace
+	return netPolList, err
```

Comment:
`err` is `nil` here, right? So, I would communicate this guarantee:
```suggestion
	return netPolList, nil
```